CMakeLists.txt

```diff
@@ -542,6 +624,18 @@ add_custom_target(shm_clean
 
 # phony target for clearing all sysrepo data
 add_custom_target(sr_clean
+    # flush all sysrepo related data from mongo
+    COMMAND "mongosh" "--eval" "use sr_startup" "--eval" "\"db.dropDatabase()\""
+                      "--eval" "use sr_running" "--eval" "\"db.dropDatabase()\""
+                      "--eval" "use sr_candidate" "--eval" "\"db.dropDatabase()\""
+                      "--eval" "use sr_operational" "--eval" "\"db.dropDatabase()\""
+                      "--eval" "use sr_factory-default" "--eval" "\"db.dropDatabase()\""
+                      "1>/dev/null"
+    
+    # flush all sysrepo related data from redis
+    COMMAND "redis-cli" "--scan" "--pattern" "\"sr:*\"" "|" "xargs" "-d" "\"\\n\"" "-I," "redis-cli" "unlink" "\",\"" "1>/dev/null"
+    COMMAND "redis-cli" "--raw" "ft._list" "|" "grep" "-x" "-G" "\"^sr:.*\"" "|" "xargs" "-n" "1" "redis-cli" "ft.dropindex" "1>/dev/null"
+
```

Comment:
Needs to be conditional based on the presence of MongoDB and redis. Several targets `sr_mongo_clean`, `sr_redis_clean`, and `sr_repo_clean` may be created and add into `sr_clean` only those supported.

Reply:
done